app/views/govuk_publishing_components/components/_column_list.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  title ||= nil
+  title_hidden ||= false
+  items ||= []
+%>
+<% if items.any? %>
+  <% if title.present? %>
+    <h2 class="<%= title_hidden ? "govuk-visually-hidden" : "govuk-heading-m" %>"><%= title %></h2>
+  <% end %>
+
+  <ul class="govuk-list gem-c-column-list">
```

Comment:
Although we don't want to display visible numbers, it seems these lists are generally ordered deliberately. You specifically want the user to read down the column, not across the row because it's been ordered, so is ol more appropriate here?

Reply:
I see your point however I don't know if this stacks for screen reader users. I remember Anika raising a couple of WCAG fails around ol's that should be ul's because there's nothing in the content to indicate order like a number or a date. Right now based on this criteria, the 2 places where this would be used don't indicate order in their content so I feel like this could open us up to a WCAG fail. I do wonder if this component becomes more wide spread if it's worth considering which tag we use for the list depending on the situation but for now I don't know if calling this an ordered list makes the most sense.

Reply:
I disagree here - they both list their contents in a predictable order: alphabetical. Not a hill I plan to die on, but worth a mention :)

Reply:
Ahaaaa... Shows how perceptive I am. As a final act of childish defiance, I'm going to ask on the accessibility channel and get a view from the hivemind.

Reply:
After asking the accessibility channel, here are few select responses:

>I’d say in most cases, a list of items in alphabetical order would not be an ordered list because you could reorder the list without materially changing the semantic meaning

>Your list is ‘ordered’ but only for ease of browsing, not because it means something to have it ordered (like a recipe has meaning to the order and doesn't work without it)

This therefore still seems like an unordered list to me because it's not necessary to read the first topic before the 2nd to understand the list fully. I feel like the the alphabetisation is purely an enhancement. This could change drastically depending on future uses of this component of course.